test/common/schemaform/widgets/DateWidget.unit.spec.jsx

```diff
@@ -63,9 +63,11 @@ describe('Schemaform: DateWidget', () => {
     const instance = tree.getMountedInstance();
     instance.handleChange('year', 2001);
     instance.handleChange('month', '1');
-    instance.handleChange('day', '2');
```

Comment:
Went back and looked at this to refresh my memory. I'm not sure what I was running into before, but it seems to work fine if they're all strings. I'm converting to number first to remove leading zeroes, but the end result seems to be fine in either type.

In this test I'm not actually using these values, so it doesn't break anything, but I'll fix it in both places.